exampleplugins/5.2/SwagLastRegistrationsWidget/Controllers/Backend/SwagLastRegistrationsWidget.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+class Shopware_Controllers_Backend_SwagLastRegistrationsWidget extends Shopware_Controllers_Backend_ExtJs
+{
+    /**
+     * Return the last registered users with an offset if it is defined
+     */
+    public function getLastRegistrationsAction()
+    {
+        $start = (int) $this->Request()->getParam('start');
+        $limit = (int) $this->Request()->getParam('limit');
+
+        $select = "
+            SELECT
```

Comment:
use DBAL query builder here and only add offset and limit if the parameters are set